src/index.ts

```diff
@@ -1,4 +1,4 @@
-export type Point = [number, number];
+export type Point = readonly [number, number];
```

Comment:
this is a potential breaking change if host apps import this type and are mutating the points somewhere (or their own APIs don't mark the point params as readonly)